gensim/models/lda_dispatcher.py

```diff
@@ -74,7 +74,7 @@
 from gensim.models.lda_worker import LDA_WORKER_PREFIX
 
 
-logger = logging.getLogger("gensim.models.lda_dispatcher")
+logger = logging.getLogger(__name__)
```

Comment:
This is not a good idea in CLI scripts (`__name__` will be just `__main__` there).

Reply:
fixed: ee87342